doc/man3/pbs_statsched.3B

```diff
@@ -47,7 +47,7 @@
 .sp
 .B void pbs_statfree(\^struct batch_status *psj\^)
 .SH DESCRIPTION
-Issue a batch request to obtain the status of PBS scheduler.
+Issue a batch request to obtain the status of (all) PBS scheduler(s).
```

Comment:
Remove the () around all.

Reply:
having () around all implies for multiple PBS schedulers. I say we keep the () around all

Reply:
() means a parenthetical remark.  In this case, all is part of the sentence.  Please remove them.

Reply:
removed the ()